phpstorm-url-handler

```diff
@@ -7,15 +7,19 @@
 # @license GPL
 # @author Stefan Auditor <stefan.auditor@erdfisch.de>
 
-arg=${1}
+urldecode() { : "${*//+/ }"; echo -e "${_//%/\\x}"; }
+arg=$(urldecode "${1}")
 pattern=".*file(:\/\/|\=)(.*)&line=(.*)"
+find="/shared/httpd/"
+replace="/home/michael/public_html/";
```

Comment:
Isn't the path mapping something, that should be handled on the application side, that builds the links?

Reply:
possibly and some apps might already have this. the software i usually develop in for web applications, doesn't have it, so i added it for my own use, but decided to keep it for this PR in case anyone else needed it.

Reply:
I see. Maybe you can also make a note in the README.md so that the user is aware of such functionality.

Reply:
added.